docs/pages/faq.rst

```diff
@@ -0,0 +1,61 @@
+Frequently Asked Question
+=========================
+
+
+Why doesn't :class:`~mcstatus.server.BedrockServer` have an async :meth:`~mcstatus.server.MCServer.lookup` method?
+------------------------------------------------------------------------------------------------------------------
+
+.. this title looks ok in rendered docs
+
+Bedrock servers do not do some DNS lookups as this do Java servers, lookup in
+:class:`~mcstatus.server.BedrockServer` just parses IP:port strings and adds
+standard port if it is not specified. So there is no need for async alternative.
```

Comment:
```suggestion
With Java servers, to find the server, we sometimes end up performing an SRV DNS 
lookup. This means making a request to your DNS server and waiting for an answer,
making that lookup a blocking operation (during which other things can be done).

However with Bedrock servers, no such lookups are required (bedrock doesn't 
support SRV records), and so there is no blocking I/O operation being made, that
would justify having an async version. 

In fact, all that the bedrock lookup does is parsing the ``host:port`` address, and 
obtaining the ``host`` and ``port`` parts out of it (with some error handling, and 
support for default ports).
```
Might be worth linking to some docs explaining what SRV records are. Alternatively, we could also just have a tip/note block with a quick explanation, like this one:

```
An SRV record allows the server to have an address like: `hypixel.net`, that points to
a some specified IP/Host and port, depending on this record.

That way, even if the server is hosted on a non-standard port (other than 25565, say 8855), 
you won't need to use `myserver.com:8855`, since the port number will simply be stored
in the SRV record, so people can still connect simply with `myserver.com`.

On top of that, it also allows to specify a different IP/Host, which means you don't need to
use the same server to run both the website, and the minecraft server. Instead, the SRV
record can simply point to a different IP/Host address (like `mc.hypixel.net`, or `209.222.114.112`).
```

Note that since I'm just writing this suggestion from the github UI, the text wrapping is certainly incorrect. Also, you might want to add some references to other objects I mentioned, if needed.

Reply:
The explanation itself is good, but I do not know where to put it. Like, "What is SRV?" is not the kind of question about mcstatus or related to mcstatus. Plus, it's not kind of frequently asked question, so I'm unsure whether we should document it. Shouldn't be enough to mention that "you should use lookup if you have a domain, if not - this is quite useless for use". I maybe will mention it in quick start docs, or other part of docs.

Reply:
Added that code suggestion in [last force-push](https://github.com/py-mine/mcstatus/compare/9086ff3d9484d8cde97d4439741329a1a487017d..f71fea6b2261db29714c612dcf737c7c75465482), thanks!

Reply:
Well, I was thinking about just including it in a `.. note` block, but I don't necessarily mind it too much if it's not included at all. However I don't think we should put it in the quick start, it's way too complex and not really necessary for end users to know. The only place that would make sense is as a part of this FAQ question, and even that's slightly weird, which is why I mentioned that we could also instead just include a link to a similar explanation somewhere.

Reply:
Related: https://documentation.divio.com/